.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's put the cache folder to ignore list

Reply:
I fixed.